src/codeenginestudio/thuy/Exercise4A.java

```diff
@@ -0,0 +1,26 @@
+package codeenginestudio.thuy;
+
+import java.util.*;
+
+public class Exercise4A {
+   public static void main(String[] args){
+
+       String[] listA = {"a", "b", "c", "d"};
+       String[] listB = {"d", "e"};
+       Set<String> ouput = findDuplicate(listA,listB);
+       System.out.println("Ouput: " + ouput);
+
+   }
+
+    private static Set<String> findDuplicate(String[] listA, String[] listB) {
+        Set<String> duplicates = new LinkedHashSet<>();
+        Set<String> uniques = new HashSet<>(Arrays.asList(listA));
+
+        for (String t : listB) {
+            if (!uniques.add(t)) {
```

Comment:
What is the meaning here? Please help me explain it.

Reply:
OK.. I know HashSet collection can not contain same value so I create a HashSet and add listA to this HashSet.
I loop listB and compare the condition, if listB[i] is added to HashSet it mean listB[i] doesnt duplicate element